src/modules/common/components/Caption.styles.js

```diff
@@ -1,5 +1,7 @@
 import styled from 'styled-components'
 
+// fixme: H1 is connected to redux state -> not a plain old component
```

Comment:
It's not connected to the redux-store, but it uses a themeProvider (provided by styled-components)...
Which makes sense, because every visual component shoud be able to access the theme